src/api/cf.nim

```diff
@@ -32,6 +33,10 @@ converter toCfModFile(json: JsonNode): CfModFile =
   result.name = json["displayName"].getStr()
   result.downloadUrl = json["downloadUrl"].getStr()
   result.gameVersions = json["gameVersion"].getElems().map((x) => x.getStr().Version)
+  result.dependencies = @[]
+  for depends in json["dependencies"].getElems():
+    if depends["type"].getInt() == 3:
+      result.dependencies.add(depends["addonId"].getInt())
```

Comment:
Nim has a nifty [`collect`](https://nim-lang.org/docs/sugar.html#collect.m%2Cuntyped%2Cuntyped) macro for that :P